test/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 7,
+    "sourceType": "module"
+  },
```

Comment:
a bit tricky on this.. conflicts between Airbnb's config, Okonoet's config, `src/`, and `test/`